src/CommunityToolkit.Maui.Core/Views/Popup/PopupExtensions.macios.cs

```diff
@@ -110,25 +111,47 @@ public static void SetLayout(this MauiPopup mauiPopup, in IPopup popup)
 				Microsoft.Maui.Primitives.LayoutAlignment.Center => frame.GetMidX(),
 				_ => 0f
 			};
-
-			if (mauiPopup.PopoverPresentationController is null)
-			{
-				throw new InvalidOperationException($"{nameof(mauiPopup.PopoverPresentationController)} Cannot Be Null");
-			}
-
 			mauiPopup.PopoverPresentationController.SourceRect = new CGRect(originX, originY, 0, 0);
 			mauiPopup.PopoverPresentationController.PermittedArrowDirections = 0;
 		}
 		else
 		{
-			if (mauiPopup.PopoverPresentationController is null)
-			{
-				throw new InvalidOperationException($"{nameof(mauiPopup.PopoverPresentationController)} Cannot Be Null");
-			}
-
 			var view = popup.Anchor.ToPlatform(popup.Handler?.MauiContext ?? throw new InvalidOperationException($"{nameof(popup.Handler.MauiContext)} Cannot Be Null"));
 			mauiPopup.PopoverPresentationController.SourceView = view;
 			mauiPopup.PopoverPresentationController.SourceRect = view.Bounds;
 		}
+
+		mauiPopup.PopoverPresentationController.PopoverBackgroundViewType = typeof(TransparentPopoverBackgroundView);
+	}
+
+	class TransparentPopoverBackgroundView : UIPopoverBackgroundView
+	{
+		public TransparentPopoverBackgroundView(IntPtr handle) : base(handle)
+		{
+			BackgroundColor = Colors.Transparent.ToPlatform();
+			Alpha = 0.0f;
+		}
+
+		public override NFloat ArrowOffset { get; set; }
+
+		public override UIPopoverArrowDirection ArrowDirection { get; set; }
+
+		[Export("arrowHeight")]
+		static new float GetArrowHeight()
```

Comment:
What are those arrow methods? Is the arrow that appears when the popup is anchored to a view?

Reply:
This is a requirement of UIPopoverBackgroundView to add these methods.
They used to create absolutely custom background (e.g. add image).